tests/adiabatic_boundary.prm

```diff
@@ -7,81 +7,75 @@ set Use years in output instead of seconds = true
 set End time                               = 0
 set Nonlinear solver scheme                = single Advection, iterated Stokes
 set Nonlinear solver tolerance             = 1e-4
-set Max nonlinear iterations               = 6 
-
+set Max nonlinear iterations               = 10
 set Pressure normalization                 = surface
 set Surface pressure                       = 0
 
+subsection Formulation
+  set Formulation = Boussinesq approximation
+end
+
 subsection Geometry model
-  set Model name = ellipsoidal chunk
-  
-  subsection Ellipsoidal chunk
-   set  NE corner = 131:-28
-   set  NW corner = 109:-28
-   set  SW corner = 109:-41
-   set  SE corner =  
+  set Model name = chunk
+
+  subsection Chunk
+    set Chunk minimum longitude = 35
+    set Chunk maximum longitude = 55
+    set Chunk minimum latitude = -28
+    set Chunk maximum latitude = -8
+    set Chunk inner radius = 5878137
+    set Chunk outer radius = 6378137
   end
-end   
+end
 
-## The input ascii data file is formatted as 3 columns of 'latitude', 'longitude', and 'depth',
-# where 'depth' is in kilometers and represents that depth of an initial temperature of 1673.15 K.
-# The latitudinal and longitudinal bounds of the ascii input data file needs to be at least 1 degree
-# wider than the bounds you use to define the ellipsoid chunk geometry model. 
-   
 subsection Initial temperature model
   set Model name = adiabatic boundary
-  subsection Adiabatic boundary
-    set Isotherm depth filename = adiabatic_boundary.txt
-  end
+#  subsection Ascii data model
+#    set Data file name = adiabatic_boundary.txt
+#  end
 end
 
 
 subsection Boundary temperature model
-  set List of model names = box
-
-  subsection Spherical constant
-    set Inner temperature = 4273 # default: 6000
-    set Outer temperature = 273.15 # default: 0
+  set List of model names = initial temperature
+  subsection Initial temperature
   end
 end
 
 # The parameters below this comment were created by the update script
 # as replacement for the old 'Model settings' subsection. They can be
 # safely merged with any existing subsections with the same name.
```

Comment:
Not directly related to your PR, but please remove the three comment lines above. They are leftovers from our conversion of all tests for ASPECT 2.0.0.